src/components/ShowsPage.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import shows from "../shows.json";
+import { Show } from "./Show";
+import { displayShow } from "./showUtls";
+import { ShowInfo } from "../showsInfo";
+
+// recieve shows state from app
+// app will handle show selection (sending state to shows & ep pages)
+
+// onClick change show state (pass handle click to Show component)
+// css
+
+interface ShowsPageProps {
+    clickedShow: (selection: ShowInfo) => void;
+    displayShowPage: (display: boolean) => void;
+}
+
+export function ShowsPage(props: ShowsPageProps): JSX.Element {
+    const [typedSearch, setTypedSearch] = useState("");
+    const [dropdownShow, setDropdownShow] = useState<ShowInfo>();
```

Comment:
Consider change naming from dropdown show to selected show => purely because you have a function handleShowSelection and than it would be self-explanatory what it does.